gradle.properties

```diff
@@ -14,7 +14,7 @@ kotlin.js.ir.output.granularity=whole-program
 # Temporarily force IDEs to produce a Build Scan
 systemProp.org.gradle.internal.ide.scan=true
 # If you're experimenting with changes and don't want to update the verification file right away, please change the mode to "lenient" (not "off")
-org.gradle.dependency.verification=strict
+org.gradle.dependency.verification=lenient
```

Comment:
how to solve this please?

Reply:
See our documentation on dependency verification:
https://docs.gradle.org/current/userguide/dependency_verification.html#verifying-dependencies

Reply:
thx.